ovirt_imageio/_internal/backends/http.py

```diff
@@ -325,10 +340,17 @@ def seek(self, n, how=os.SEEK_SET):
         return self._position
 
     def size(self):
-        # We have 2 bad options:
-        # - Get last extent, may be slow, and may not be neded otherwise.
-        # - Emulate HEAD request, logging tracebacks in the remote server.
-        # Getting extents is more polite, so lets use it if we can.
+        """
+        Return backend size in bytes.
+
+        With newer server (imageio >= 2.3) reporting the virtual size the size
```

Comment:
The version here should probably be  >= 2.5 - the version including this feature.